specification/containerinstance/resource-manager/Microsoft.ContainerInstance/preview/2024-11-01-preview/containerInstance.json

```diff
@@ -1756,6 +1746,24 @@
         "securityContext": {
           "description": "The container security properties.",
           "$ref": "#/definitions/SecurityContextDefinition"
+        },
+        "configMap": {
+          "description": "The config map.",
```

Comment:
>ConfigMap [](http://example.com/codeflow?start=5&length=9)

for next reviewer, this is backfilling from 2024-05-01-preview api versions, that is missed in 2024-09-01-preview and 2024-10-01-preview